ftva_lab_data/tests.py

```diff
@@ -202,11 +202,11 @@ def setUp(self):
 
         # Create enough SheetImport objects to require pagination
         # (i.e. more than the default page size of 10)
```

Comment:
The comment should be updated to reflect this is now to test pagination with more than the max value of `items_per_page`.